src/standalone/SignaturePad/index.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { useState } from "react";
+import { makeStyles, Tooltip } from "@material-ui/core";
+import { Info as InfoIcon, Edit as EditIcon } from "@material-ui/icons";
+import SignPadDialog from "./SignPadDialog";
+import ccI18n from "../../i18n";
+
+export interface SignaturePadCanvasProps {
+	infoText?: React.ReactNode;
+	canvasProps?: React.CanvasHTMLAttributes<HTMLCanvasElement>;
+	clearOnResize?: boolean;
+	penColor?: string;
+	disabled?: boolean;
+	setSignature?: string;
+	getSignature?: (imageURL: string) => void;
+}
+
+const useClasses = makeStyles((theme) => ({
+	signPadDiv: {
+		position: "relative",
+		cursor: "pointer",
+		borderBottom: "1px dotted",
+		height: 150,
+		width: 400,
+		color: theme.palette.grey[700],
+		display: "inline-block",
+	},
+	textDiv: {
+		position: "absolute",
+		display: "inline-block",
+		bottom: 0,
+		left: 5,
+	},
+	imageDiv: {
+		display: "inline-block",
+		margin: "0px 20px",
+	},
+	signText: {
+		display: "inline-block",
+		marginLeft: 10,
+	},
+	infoDiv: {
+		position: "absolute",
+		right: 5,
+		bottom: 0,
+	},
+}));
+
+const SignaturePadCanvas = (props: SignaturePadCanvasProps) => {
+	const {
+		setSignature,
+		getSignature,
+		disabled,
+		infoText,
+		clearOnResize,
+		canvasProps,
+		penColor,
+	} = props;
+	const classes = useClasses();
+	const [imageURL, setImageURL] = useState(setSignature || "");
```

Comment:
these components should be controllable via props => shouldn't store their values in their own state